src/java.base/share/classes/java/lang/invoke/InnerClassLambdaMetafactory.java

```diff
@@ -380,9 +380,9 @@ public Void run() {
             Lookup lookup;
             if (useImplMethodHandle) {
                 lookup = caller.defineHiddenClassWithClassData(classBytes, implementation, !disableEagerInitialization,
-                                                               NESTMATE, STRONG);
```

Comment:
Please remove the import for this constant as well. It seems to be unused now.

Reply:
Will do (and will updated the copyright year as well :)